lib/apiForSheets.js

```diff
@@ -103,10 +107,11 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
         'Welcome to Oppia! Please could you ' +
         'follow the instructions ' +
         linkResult +
-        " to get started? You'll need to do " +
-        'this before we can accept your PR. I am closing this PR for now. ' +
-        'Feel free to re-open it once you are done with the above ' +
-        'instructions. Thanks!',
+        " to get started with oppia and "  +  linkOppiaAndroid +
+        " to get started with oppia-android? You'll need to do this " +
+        "before we can accept your PR. I am closing this PR for now. " +
+        "Feel free to re-open it once you are done with the above " +
+        "instructions. Thanks!",
```

Comment:
Can you use an if statement to check if the PR is from the oppia android repo and then respond with different comments for the repos?

Reply:
addressed in latest commit.